packages/api/src/external/location/address.ts

```diff
@@ -0,0 +1,41 @@
+import { Address } from "@metriport/api-sdk";
+import {
+  getCoordinatesFromLocation,
+  makeLocationClient,
+} from "@metriport/core/external/aws/location";
+import { Config } from "../../shared/config";
+import { Coordinates } from "@metriport/core/external/aws/location";
+
+type AddressWithoutCountry = Omit<Address, "country"> & { country?: string | undefined };
+
+/**
+ * Geocodes a list of addresses using Amazon Location Services.
+ * @param addresses
+ * @returns
+ */
+export async function getCoordinates(addresses: AddressWithoutCountry[]): Promise<Coordinates[]> {
+  const indexName = Config.getPlaceIndexName();
+  const awsRegion = Config.getAWSRegion();
+  const client = makeLocationClient(awsRegion);
+
+  const coords = [];
+
+  for (const address of addresses) {
+    const addressText =
+      address.addressLine1 + ", " + address.city + ", " + address.state + " " + address.zip;
```

Comment:
use template literals to improve code readability:
`const addressText = `${address.addressLine1}, ${address.city}, ${address.state} ${address.zip}`;`